dll/win32/comctl32/listview.c

```diff
@@ -8749,6 +8749,13 @@ static DWORD LISTVIEW_SetExtendedListViewStyle(LISTVIEW_INFO *infoPtr, DWORD mas
         LISTVIEW_UpdateScroll(infoPtr);
     }
 
+#ifdef __REACTOS__
+    if((infoPtr->dwLvExStyle ^ old_ex_style) & LVS_EX_SNAPTOGRID)
```

Comment:
```suggestion
    if ((infoPtr->dwLvExStyle & LVS_EX_SNAPTOGRID) > (old_ex_style & LVS_EX_SNAPTOGRID))
```

Reply:
Thanks!  I'll get this updated in the PR tonight.